### PR TITLE
[Hightouch] Add support for `schedulePaused`

### PR DESCRIPTION
**Pull request by karan-batavia**

### CODE REVIEW

File: src/main/scala/ai/privado/utility/SQLParser.scala [hunk 1]
```diff
@@ -62,11 +62,8 @@ object SQLParser {
       case plainSelect: PlainSelect if plainSelect.getFromItem.isInstanceOf[Table] =>
         val sqlTable = createSQLTableItem(plainSelect.getFromItem.asInstanceOf[Table])
         List(SQLQuery(SQLQueryType.SELECT, sqlTable, getColumns(plainSelect, sqlTable)))
-      case plainSelect: PlainSelect if plainSelect.getFromItem.isInstanceOf[ParenthesedSelect] =>
-        createSQLNodesForSelect(List(plainSelect.getFromItem.asInstanceOf[ParenthesedSelect].getSelect))
-          .getOrElse(List.empty[SQLQuery])
       case parenthesedSelect: ParenthesedSelect =>
-        createSQLNodesForSelect(List(parenthesedSelect.getSelect)).getOrElse(List.empty[SQLQuery])
+        createSQLNodesForSelect(List(parenthesedSelect.getPlainSelect)).getOrElse(List.empty[SQLQuery])
```
Comment (khemrajrathore) [inline src/main/scala/ai/privado/utility/SQLParser.scala:66]: Does using getSelect result in some problem?

**Review by khemrajrathore — DISMISSED** [review]
Lgtm, do proper testing locally

File: src/main/scala/ai/privado/utility/SQLParser.scala (hunk 1 above)
Comment (karan-batavia) [reply src/main/scala/ai/privado/utility/SQLParser.scala:66]: I pulled in dev, which had this code.

**Review by pandurangpatil — CHANGES_REQUESTED** [review]
@karan-batavia create a new PR with the changes applied on a freshly created branch from `dev`.

cc: @khemrajrathore

**Pull request closed without merging**